release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
This stub looks good and closely matches what was done for 4.19: https://raw.githubusercontent.com/openshift/openshift-docs/refs/heads/main/release_notes/ocp-4-19-release-notes.adoc